.github/workflows/build.yaml

```diff
@@ -75,14 +75,15 @@ jobs:
 
       - name: Install buildtools (MacOS)
         if: matrix.os == 'macOS-latest'
-        run: brew install automake
+        run: |
+          brew install automake libtool
 
       - name: Install libsecp256k1 (Unix)
```

Comment:
I wonder if we want to handle this with nix? tbh I'm not actually a nix expert but afaict that's the way it's handled [here](https://github.com/haskell-bitcoin/libsecp256k1-haskell/blob/master/stack.yaml#L2-L5). Not blocking but wanted to solicit some feedback here as it could alleviate you from having to do the cmake incantations below.

Reply:
I'm in favor of using nix here, although I believe that nix doesn't support windows yet without something like WSL. Having the nix configs in the stack.yaml might make it harder for contributor without a nix installation to build. If we're happy to, I can add it to the stack.yaml or we can define a separate one with the nix config as pass the file as an argument to all the stack commands in the build.

Reply:
This is a good point, and enough for me to put on hold any changes that move us towards nix. We can revisit in the future if someone is championing it.